discovery/gossiper.go

```diff
@@ -3250,9 +3243,7 @@ func (d *AuthenticatedGossiper) handleAnnSig(nMsg *networkMsg,
 	// after some number of confirmations after channel was registered in
 	// bitcoin blockchain. Therefore, we check if the proof is mature.
 	d.Lock()
-	premature := d.isPremature(
-		ann.ShortChannelID, d.cfg.ProofMatureDelta, nMsg,
-	)
+	premature := d.isPremature(ann.ShortChannelID, nMsg)
```

Comment:
I wonder if we also check this for Alias-Channel, shouldn't we use the Real-SCID for zero-conf channels here ?